lib/msal-core/src/utils/UrlUtils.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'd argue you don't need to include `Utils` in the name, since its in the path already: `utils/Url.ts`

Reply:
I'd prefer to name files what they are, regardless of folder. It is easier to read in stacktraces and such, but I'm open to this if the team feels it is more concise.